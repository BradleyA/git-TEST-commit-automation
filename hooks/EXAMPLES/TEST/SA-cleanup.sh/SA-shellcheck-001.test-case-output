
In /home/uadmin/github/BradleyA/git-TEST-commit-automation/hooks/EXAMPLES/SA-cleanup.sh line 53:
for k in $(ls -1 SA-*) ; do
         ^-- SC2045: Iterating over ls output is fragile. Use globs.

