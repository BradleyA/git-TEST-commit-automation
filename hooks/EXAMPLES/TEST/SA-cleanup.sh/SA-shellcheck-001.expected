
In SA-shellcheck-001 line 55:
for k in $(ls -1 SA-*) ; do
         ^-- SC2045: Iterating over ls output is fragile. Use globs.

